runtime/ts/manifest.ts

```diff
@@ -115,7 +115,7 @@ export class Manifest {
   storeTags: Map<StorageProviderBase, string[]> = new Map();
   private _fileName: string|null = null;
   private nextLocalID = 0;
-  private _id: Id;
+  private readonly _id: Id;
```

Comment:
consider converting this to `public readonly id: Id;` with no getter.

Reply:
Will look into a followup on this.  There's some logic in the getter:

```
  get id() {
    if (this._meta.name) {
      return Id.newSessionId().fromString(this._meta.name);
    }
    return this._id;
  }
```